onnx/backend/test/cpp/driver/test_driver.cc

```diff
@@ -206,6 +198,11 @@ ResolvedTestCase LoadSingleTestCase(const UnsolvedTestCase& t) {
       std::string input_data;
       ONNX_NAMESPACE::ValueInfoProto input_info;
       LoadSingleFile(input_file, input_data);
+      if (test_data_counter >= st.model_.graph().input().size()) {
```

Comment:
nip: shall we take into consideration on feed name and check all model's required inputs get mapped to from the feed?

Reply:
I don't quite understand -- what kind of feed name you were saying? If you were saying input/output names, then ONNX uses naming like `input_X.pb` and `output_X.pb` for input/output files. If added .pb files do not follow this naming, other CIs will catch this kind of error (during test generation). For here, I believe it will simply fail due to unfound data. I added the size check here, because previously wrong test data size will lead to segmentation fault (access out of range), which is not acceptable.